addons/compat_spe/stringtable.xml

```diff
@@ -3,57 +3,93 @@
     <Package name="compat_spe">
         <Key ID="STR_ACRE_compat_spe_RadioSetShort">
             <English>Radio</English>
+            <German>Funkgerät</German>
+            <Italian>Radio</Italian>
         </Key>
         <Key ID="STR_ACRE_compat_spe_RadioSet">
             <English>Radio Set</English>
+            <German>Funkkanäle</German>
+            <Italian>Canali radiofonici</Italian>
         </Key>
         <Key ID="STR_ACRE_compat_spe_RadioSet1_Short">
             <English>A-Set</English>
+            <German>A-Kanäle</German>
+            <Italian>Canali A</Italian>
         </Key>
         <Key ID="STR_ACRE_compat_spe_RadioSet1_US">
             <English>Radio Set - Able</English>
+            <German>Kanäle - Able</German>
+            <Italian>Canali - In grado</Italian>
```

Comment:
`Able` here is just a callsign, no point in translating it to a phrase, better to just leave it as-is like was done with the other Sets.
```suggestion
            <Italian>Canali - Able</Italian>
```